ob-glsl.el

```diff
@@ -1,4 +1,23 @@
+;;; ob-glsl.el --- description pending -*- lexical-binding: t; -*-
+
+(defcustom ob-glsl-make-command "cmake -G \"Ninja\" . && ninja"
+  "build command used when compiling ob-glsl"
+  :type 'string
+  :group 'ob-glsl)
+
 (require 'ob)
+
+;; setup
+(defun ob-glsl-compile ()
+  (let ((default-directory (file-name-directory load-file-name)))
+	(shell-command ob-glsl-make-command)
+	(load-file
+	 (concat default-directory
+			 (car (directory-files default-directory nil "^ob-glsl-module\\.\\(so\\|dll\\)$"))))
```

Comment:
on macos the file extension is "dylib"

Reply:
Might want to update the README to include that information in the meantime. Right now it only specifies `.so` and `.dll`.

Reply:
I fixed the README file.